notebooks/5_CUR.ipynb

```diff
@@ -162,10 +162,12 @@
     "For the $j^{th}$ column, the importance score is given by\n",
     "\n",
     "\\begin{equation}\n",
-    "\\pi_j = \\sum_i^k (U_C)_{ij}^2,\n",
+    "\\pi_j = \\sum_j^k (U_C)_{ij}^2,\n",
     "\\end{equation}\n",
     "\n",
-    "where $k$ is again the target rank of the new matrix and $(U_C)_{ij}$ is the value in the $i^{th}$ row and $j^{th}$ column of $\\mathbf{U}_C$."
+    "where $k$ is again the target rank of the new matrix and $(U_C)_{ij}$ is the value in the $i^{th}$ row and $j^{th}$ column of $\\mathbf{U}_C$.\n",
+    "\n",
+    "Note that `sps.linalg.svds` actually returns $U_C^T$ having right singular vectors as rows. So we use `sum (axis = 0)` below."
```

Comment:
This will not be necessary when you incorporate the new code.

Reply:
Okay, I will remove it.